addons/product/product_view.xml

```diff
@@ -211,17 +211,66 @@
+                            <field name="image_medium" widget="image" class="oe_avatar oe_left"/>
+                            <div class="oe_title" style="width: 390px;">
+                                <label class="oe_edit_only" for="name" string="Product Name"/>
+                                <h1><field name="name" class="oe_inline"/></h1>
+                            </div>
+                        </div>
+                        <group>
+                            <p>All general settings about this product are managed on <button name="open_product_template" type="object" string="the product template" class="oe_link"/></p>
+                        </group>
+                        <group string="Attributes">
+                            <field name="attribute_value_ids" widget="many2many_tags"/>
+                        </group>
+                        <group>
+                            <group name="codes" string="Codes">
+                                <field name="barcode"/>
+                                <field name="default_code"/>
+                            </group>
+                            <group>
+                                <field name="active"/>
+                            </group>
+                        </group>
+                        <group>
+                            <group name="pricing" string="Pricing">
+                                <field name="lst_price"/>
+                                <field name="standard_price"/>
+                            </group>
+                            <group name="weight">
+                                <group name="weight" string="Weights">
+                                    <field digits="(14, 3)" name="volume"/>
+                                    <field digits="(14, 3)" name="weight"/>
+                                    <field digits="(14, 3)" name="weight_net"/>
+                                </group>
+                            </group>
+                        </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
         <record id="product_variant_action" model="ir.actions.act_window">
             <field name="name">Product Variants</field>
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">product.product</field>
-            <field name="view_mode">tree,form,kanban</field>
-            <field name="view_type">form</field>
             <field name="context">{'search_default_product_tmpl_id': [active_id], 'default_product_tmpl_id': active_id}</field>
-            <field name="search_view_id" ref="product_search_form_view"/>
-            <field name="view_id" eval="False"/> <!-- Force empty -->
-            <field name="domain" eval="False"/> <!-- Force empty -->
-            <field name="help" type="html">
+            <field name="search_view_id" ref="product_search_form_view"/>             
+            <field name="view_ids" 
```

Comment:
Seems nicer than defining unnecessary xml_ids. Al and I approve.

Just add spaces :postal_horn:  !